### PR TITLE
Game Update Notifications

### PR DESCRIPTION
**Pull request by Whomstt**
Integrated notification service with game service to provide email notifications providing updates to players assigned to games about their tournament.

### CODE REVIEW

File: game-service/src/main/java/com/lms/gameservice/service/GameUpdateScheduler.java
Comment (LiamMoMurray) [inline game-service/src/main/java/com/lms/gameservice/service/GameUpdateScheduler.java:1]: Would it be possible to refactor changes inside the methods here to a new file in the services tab called NotificationServiceClient?

**Review by LiamMoMurray — CHANGES_REQUESTED** [review]
comment on GameScheduler

**Review by LiamMoMurray — APPROVED** [review]
Looks good, will refactor later

**Review by olanhealy — APPROVED** [review]
Sorry forgot to submit earlier, all good to me

**Pull request merged**